plugins/src/test/java/com/qubole/quark/plugins/jdbc/JdbcFactoryValidationTest.java

```diff
@@ -33,7 +33,7 @@ public void testGenericDb() throws QuarkException {
   public void testValidProps() throws QuarkException {
     Map<String, Object> props = new HashMap<>();
     props.put("type", "mysql");
-    props.put("url", "jdbc:quark:test");
+    props.put("url", "fatjdbc:quark:test");
```

Comment:
Oh yea - that doesn't look right. Should it be 'jdbc:quark:fat:' ?
On Tue, 8 Mar 2016 at 19:17, amoghmargoor <notifications@github.com> wrote:

> In
> plugins/src/test/java/com/qubole/quark/plugins/jdbc/JdbcFactoryValidationTest.java
> <https://github.com/qubole/quark/pull/32#discussion_r55359978>:
>
> > @@ -33,7 +33,7 @@ public void testGenericDb() throws QuarkException {
> >    public void testValidProps() throws QuarkException {
> >      Map<String, Object> props = new HashMap<>();
> >      props.put("type", "mysql");
> > -    props.put("url", "jdbc:quark:test");
> > +    props.put("url", "fatjdbc:quark:test");
>
> Have you changed the connect string for fat jar from jdbc:quark to
> fatjdbc:quark ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/qubole/quark/pull/32/files#r55359978>.
>